packages/contracts/test/contracts/OVM/accounts/OVM_ECDSAContractAccount.spec.ts

```diff
@@ -74,9 +80,38 @@ describe('OVM_ECDSAContractAccount', () => {
     Mock__OVM_ExecutionManager.smocked.ovmADDRESS.will.return.with(
       await wallet.getAddress()
     )
+    Mock__OVM_ExecutionManager.smocked.ovmEXTCODESIZE.will.return.with(1)
     Mock__OVM_ExecutionManager.smocked.ovmCHAINID.will.return.with(420)
     Mock__OVM_ExecutionManager.smocked.ovmGETNONCE.will.return.with(100)
-    Mock__OVM_ExecutionManager.smocked.ovmCALL.will.return.with([true, '0x'])
+    Mock__OVM_ExecutionManager.smocked.ovmCALL.will.return.with(
+      (gasLimit, target, data) => {
+        if (target === '0x4200000000000000000000000000000000000006') {
```

Comment:
nit: getting this from the `predeploys` export would improve readability on this test